### PR TITLE
Update to libaio from @tinywrkb

### PR DESCRIPTION
**Pull request by RobLoach**
Recommended change from @tinywrkb from https://github.com/flathub/org.libretro.RetroArch/issues/186#issuecomment-876682392 .

### CODE REVIEW

**Comment by tinywrkb** [comment]
Tested locally and these are the needed changes

```
diff --git a/libaio/libaio-0.3.112.json b/libaio/libaio-0.3.112.json
index c5942d6..0491b7a 100644
--- a/libaio/libaio-0.3.112.json
+++ b/libaio/libaio-0.3.112.json
@@ -1,6 +1,9 @@
 {
   "name": "libaio",
   "no-autogen": true,
+  "make-install-args": [
+    "prefix=/app"
+  ],
   "sources": [
     {
       "type": "archive",
```

p.s. Are these really needed? Tou already have those libs in the runtime, though libpng is newer in the runtime.

```
        "libpng/libpng-1.6.35.json",
        ...
        "udev/udev-248.json",
        "libbz2/libbz2-1.0.8.json",

```

**Comment by tinywrkb** [comment]
This is also working

```
diff --git a/libaio/libaio-0.3.112.json b/libaio/libaio-0.3.112.json
index c5942d6..0491b7a 100644
--- a/libaio/libaio-0.3.112.json
+++ b/libaio/libaio-0.3.112.json
@@ -1,6 +1,9 @@
 {
   "name": "libaio",
   "no-autogen": true,
+  "make-install-args": [
+    "prefix=${FLATPAK_DEST}"
+  ],
   "sources": [
     {
       "type": "archive",
```

File: libaio/libaio-0.3.112.json [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
-        "url": "https://pagure.io/libaio.git",
-        "tag": "libaio-0.3.112",
-        "commit": "d025927efa75a0d1b46ca3a5ef331caa2f46ee0e"
-      }
-    ]
-  }
-  
\ No newline at end of file
+  "name": "libaio",
+  "no-autogen": true,
```
Comment (RobLoach) [inline libaio/libaio-0.3.112.json:3]:
``` suggestion
  "no-autogen": true,
  "make-install-args": [
    "prefix=/app"
  ],
```

File: libaio/libaio-0.3.112.json [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
+  "make-install-args": [
+    "prefix=/app"
```
Comment (RobLoach) [inline libaio/libaio-0.3.112.json:5]:
``` suggestion
    "prefix=${FLATPAK_DEST}"
```

**Comment by tinywrkb** [comment]
bot, build org.libretro.RetroArch

**Pull request merged**